nodebuilder/fraud/opts.go

```diff
@@ -0,0 +1,15 @@
+package fraud
+
+import (
+	"go.uber.org/fx"
+
+	"github.com/celestiaorg/celestia-node/fraud"
+)
+
+// WithMetrics enables metrics exporting for the node.
+func WithMetrics(enable bool) fx.Option {
+	if !enable {
+		return fx.Options()
+	}
```

Comment:
This pattern is common and allows users to avoid the if. An example would passing value of the bool flag directly into the option. In our usage case though it's not that important so I we can remove it

Reply:
removed